PusherPlatform.xcodeproj/xcshareddata/xcschemes/PusherPlatform.xcscheme

```diff
@@ -40,6 +40,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      enableThreadSanitizer = "YES"
```

Comment:
Should this stay enabled?

Reply:
Yes, let's keep this enabled (it is enabled for Chatkit as well). It does not cost us much when running unit tests and it helps highlight issues with concurrency.